src/utils/currencyList.tsx

```diff
@@ -335,12 +335,19 @@ export function hexToCurrencySymbol(hex: string) {
 
 const convertToBlockchainUnit = (amount: number | string, ccy: Currency) => {
     const value = new BigNumberJS(amount).multipliedBy(10 ** ccy.decimals);
-    if (value.isLessThan(new BigNumberJS(1))) {
+
+    if (
+        value.isNaN() ||
+        value.isLessThan(new BigNumberJS(1)) ||
+        value.toString().includes('.')
```

Comment:
This is not a good workaround. I guess what we should do is clear the value when the deposit modal is closed.